docs/javascript/conjur-account.js

```diff
@@ -42,23 +42,48 @@ function displayAccountCredentials(email, account_id, api_key) {
   });
 }
 
+function displayError(formField, message) {
+  // Manually inserting the error here because there doesn't seem to
+  // be a way to make bootstrap-validator display an error on-demand.
+  var errorElement =
+    '<ul class="list-unstyled"> \
+      <li style="color:#a94442">' +
```

Comment:
Can we use a class instead of an explicit color hex here?

Reply:
Yes I imagine we can! Will get that hooked up.